x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/events_viewer/events_viewer.spec.ts

```diff
@@ -57,7 +57,7 @@ describe('Events Viewer', () => {
       .should('eq', 'Customize Columns');
   });
 
-  it('closes the fields browser when the user clicks outside of it', () => {
+  it.skip('closes the fields browser when the user clicks outside of it', () => {
```

Comment:
maybe wrap all the `skip` tests in one `describe.skip`

Reply:
Good suggestion :) 

For now I prefer to maintain them in that way just in case the order it is relevant.